tools/clang_tidy/lib/src/command.dart

```diff
@@ -57,6 +57,7 @@ class Command {
 
   static final RegExp _pathRegex = RegExp(r'\S*clang/bin/clang');
   static final RegExp _argRegex = RegExp(r'-MF \S*');
+  static final RegExp _sedRegex = RegExp(r'&& sed.*$');
```

Comment:
I wonder if this shouldn't just filter out anything following the first `&&`?

Reply:
done